conf/systemd/packetfence-pfqueue.service

```diff
@@ -15,8 +15,8 @@ TimeoutStopSec=60
 NotifyAccess=all
 StartLimitBurst=3
 StartLimitInterval=10
-ExecStart=/usr/local/pf/sbin/pfqueue-docker-wrapper
-ExecStop=/bin/bash -c "docker stop pfqueue ; echo Stopped"
+ExecStartPre=/bin/perl -I/usr/local/pf/lib -I/usr/local/pf/lib_perl/lib/perl5 '-Mpf::services::manager::pfqueue' -e 'pf::services::manager::pfqueue->new()->generateConfig()'
+ExecStart=/usr/local/pf/sbin/pfqueue
```

Comment:
This change is intentional, we need to have pfqueue as a non-container service for creating/modifying/deleting kernel net namespaces